website_breadcrumb/README.rst

```diff
@@ -0,0 +1,94 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+   :alt: License: LGPL-3
+
+===================
+Website Breadcrumbs
+===================
+
+This module allows you to have breadcrumbs in any page of your website.
+But if an URL is not present in a menu (as indicated in the Configuration section), no breadcrumbs will display.
+
+Configuration
+=============
+
+To configure the shown breadcrumbs, you need to:
+
+* Go to *Settings > Configuration > Website Settings > Configure website
```

Comment:
Please update the path of this configuration putting: `Go to *Website Admin > Configuration* and click on "Configure website menus"`